### DISCUSSION TITLE
Code block and tables

### DISCUSSION THREAD

**Discussion by ibahnasy** [zim-desktop-wiki/zim-desktop-wiki#2257] [Ideas]
Zim should be able to insert a code block and tables of cell sizes/count.

**Comment by introt** [comment]
Zim indeed can!

Enable the [Source View plugin](https://zim-wiki.org/manual/Plugins/Source_View.html) to insert code blocks. To create tables, enable the [Table Editor plugin](https://zim-wiki.org/manual/Plugins/Table_Editor.html). Both can be found under Edit > Preferences > Plugins.
